docs/source/tutorials/audio.ipynb

```diff
@@ -85,6 +85,11 @@
    "source": [
     "# Package installation (hidden on docs website).\n",
     "# Package versions used: tensorflow==2.9.1 speechbrain==0.5.12 tensorflow-io==0.26.0 torch==1.11.0 torchaudio==0.11.0\n",
+    "\n",
+    "# Supress outputs that may appear if tensorflow happens to be improperly installed: \n",
+    "import os \n",
+    "os.environ[\"TF_CPP_MIN_LOG_LEVEL\"] = \"3\" \n",
+    "\n",
```

Comment:
Can we not move this above `dependencies`, which is a variable all tutorial authors must declare and thus should be very close to top of the notebooks.

Reply:
I saw [here](https://www.kaggle.com/c/ventilator-pressure-prediction/discussion/274717) that we should set the tf log level before importing tensorflow to suppress the warnings, hence I move it up. 
Changed it so that the `dependencies` variable is above this log but imports are still done below.